client/my-sites/media-library/style.scss

```diff
@@ -115,3 +115,27 @@
 .media-library__list-item.is-small .media-library__list-item-file-extension {
 	display: none;
 }
+
+.media-library__videopress-nudge-container {
+	display: block;
+	overflow-y: auto;
+	position: absolute;
```

Comment:
As far as I remember I've ran into some issues when trying to position it, because of absolute positioning of some other parts of media container. It might not be necessary and I'll try to avoid it.